src/Controller/CrimeSceneController.php

```diff
@@ -14,8 +14,23 @@ public function index(): string
         return $this->twig->render('Home/index.html.twig', ['titles' => $titles]);
     }
 
+    /**
+     * Add a new item
```

Comment:
you didn't add an item ;) 
Be careful to modify the code when you copy and paste